tests/test_query.py

```diff
@@ -60,6 +60,14 @@ def _assert_bad_query(self, q):
   def test_abort(self):
     self._assert_bad_query(query.abort("aborting"))
 
+  def test_query(self):
```

Comment:
I can only see two basic tests here:
1. create a `Query` instance (with no `api_version` set) and send it to server, and asset that the echoed value contains `api_version: "2.12"`
2. call the function `query()` and assert the returned value contains `api_version: "3"`

Reply:
I `unresolved` that comment: I'm not seeing the two tests here. Am I missing something?